packages/api-plugin-fulfillment-method-pickup-store/src/index.js

```diff
@@ -0,0 +1,36 @@
+import { createRequire } from "module";
+import fulfillmentMethodPickupStorePreStartup from "./preStartup.js";
+import fulfillmentMethodPickupStoreStartup from "./startup.js";
+import { MethodStoreData } from "./simpleSchemas.js";
+import schemas from "./schemas/index.js";
+import getFulfillmentMethodsWithQuotesPickupStore from "./getFulfillmentMethodsWithQuotesPickupStore.js";
+import validateOrderMethodsstore from "./util/validateOrderMethodsstore.js";
+
+const require = createRequire(import.meta.url);
+const pkg = require("../package.json");
+
+/**
+ * @summary Import and call this function to add this plugin to your API.
+ * @param {Object} app The ReactionAPI instance
+ * @returns {undefined}
+ */
+export default async function register(app) {
+  await app.registerPlugin({
+    label: "Fulfillment Method Pickup Store",
+    name: "fulfillment-method-pickup-store",
+    version: pkg.version,
+    graphQL: {
+      schemas
+    },
+    simpleSchemas: {
+      MethodStoreData
+    },
+    functionsByType: {
+      preStartup: [fulfillmentMethodPickupStorePreStartup],
+      startup: [fulfillmentMethodPickupStoreStartup],
+      validateOrderMethods: [validateOrderMethodsstore],
+      getFulfillmentMethodsWithQuotes: [getFulfillmentMethodsWithQuotesPickupStore],
+      getFulfillmentMethodsWithQuotespickup: [getFulfillmentMethodsWithQuotesPickupStore]
```

Comment:
Just my opinion on how to passing function with key in register function.
This may look better

```suggestion
      getFulfillmentMethodsWithQuotes: {
        key: "pickup",
        handler: getFulfillmentMethodsWithQuotesPickupStore,
      }
```

or 

```
```suggestion
      getFulfillmentMethodsWithQuotes: [{
        key: "pickup",
        handler: getFulfillmentMethodsWithQuotesPickupStore,
      }]
```
This an example on promotion: https://github.com/reactioncommerce/reaction/blob/4820b35b6d6c0f9f2951f1831d0d831b4ba590f9/packages/api-plugin-promotions-coupons/src/index.js#L20

Reply:
Thanks @vanpho93  ... we were thinking on how we could fix it. Your suggestion is a very elegant and made it easy to implement

Reply:
will be implementing them in the split PRs (https://github.com/reactioncommerce/reaction/pull/6570, https://github.com/reactioncommerce/reaction/pull/6579, https://github.com/reactioncommerce/reaction/pull/6577)